src/uct/ib/base/ib_iface.c

```diff
@@ -690,30 +690,10 @@ static int uct_ib_iface_is_same_device(uct_ib_iface_t *iface,
     return 1;
 }
 
-int uct_ib_iface_is_reachable_v2(const uct_iface_h tl_iface,
-                                 const uct_iface_is_reachable_params_t *params)
-{
-    uct_ib_iface_t *iface           = ucs_derived_of(tl_iface, uct_ib_iface_t);
-    const uct_ib_address_t *ib_addr = (const void*)params->device_addr;
-    uct_iface_reachability_scope_t scope;
-
-    if (!uct_ib_iface_is_reachable(tl_iface, params->device_addr,
-                                   params->iface_addr)) {
-        return 0;
-    }
-
-    scope = UCS_PARAM_VALUE(UCT_IFACE_IS_REACHABLE_FIELD, params, scope, SCOPE,
-                            UCT_IFACE_REACHABILITY_SCOPE_NETWORK);
-
-    return (scope == UCT_IFACE_REACHABILITY_SCOPE_NETWORK) ||
-           uct_ib_iface_is_same_device(iface, ib_addr);
-}
-
-int uct_ib_iface_is_reachable(const uct_iface_h tl_iface,
-                              const uct_device_addr_t *dev_addr,
-                              const uct_iface_addr_t *iface_addr)
+int uct_ib_iface_addr_is_reachable(uct_ib_iface_t *iface,
```

Comment:
can it be static?
can we remove iface_addr?

Reply:
removed parameter, for static, it is used in gtest

Reply:
added static now since gtest is not using it anymore